### DISCUSSION TITLE
Exisiting Measures

### DISCUSSION THREAD

**Discussion by ringovski** [DaxStudio/DaxStudio#685] [Q&A]
Hi All,
Using DAX Studio version 2.8.3

I would like to troubleshoot/run existing measures from a PBIX file. I connect to the model no problem and can see the large list of existing measures. I right click and select define measure and try adding Evaluate, doesn't work, try removing define and the "MEASURE xxxxx =" so it's just Evaluate calculate(... but this doesn't work either.

It seems that DAX studio can't run existing measures without additional commands or formatting and I don't know what steps are required.

E.G.
EVALUATE
DEFINE
MEASURE ABCD = CALCULATE(COUNTROWS(Postal)
,Postal[Type] ="M'
,Postal[Source] "1"
,Postal[Status] "A" || Postal[Status] ="B"
,Postal[Type] <> "D"
,FILTER(Postal,Postal[Event] = LOOKUPVALUE(LookupParameters[Value Int], [Look Values], "ABCDEF"))
)

Thanks for any assistance.

**Comment by dgosbell** [comment]
The reason this is not working is that your syntax is not valid.

The simplest query to execute a single measure is to use the curly brace table constructor syntax:
```
EVALUATE {  [ABCD]  }
```

You can see the above and other examples at the link below:
https://daxstudio.org/tutorials/writing-dax-queries/

If you update to the current release the link above is part of the help text displayed on a blank query document
